### PR TITLE
Update database-per-tenant.md

### PR DESCRIPTION
**Pull request by carlotas19**
- Messaging refresh for our DB-per-tenant page 
- It also makes the page more scrollable

### CODE REVIEW

File: content/use-cases/database-per-tenant.md [hunk 1]
```diff
@@ -53,23 +46,42 @@ author={{
...
-## Why Neon is different
+## Database-per-user in Neon
+
+Neon is Postgres with serverless architecture. With rapid provisioning, scale-to-zero, and robust API support, you can scale database-per-user architectures without management overhead or big budgets. Just create **one project per customer** via the Neon API.
+
+### One project per customer
+
+A Neon project is the logical equivalent of an "instance" but without the management heaviness:
+
+- By creating one project per customer, each customers' data will be completely isolated.
```
Comment (danieltprice) [inline content/use-cases/database-per-tenant.md:57]:
```suggestion
- By creating one project per customer, each customer's data will be completely isolated.
```

File: content/use-cases/database-per-tenant.md [hunk 2]
```diff
@@ -53,23 +46,42 @@ author={{
...
+- You'll be able to run independent PITRs without affecting your entire fleet.
+- You can create diffeent projects in different regions to match your customers' location.
```
Comment (danieltprice) [inline content/use-cases/database-per-tenant.md:59]:
```suggestion
- You can create different projects in different regions to match your customers' locations.
```

File: content/use-cases/database-per-tenant.md [hunk 3]
```diff
@@ -53,23 +46,42 @@ author={{
...
+
+Management is simplified vs other Postgres services because,
+
+- There’s no need to provision infrastructure in advance.
+- You can scale your architecture progressively, from a few tenants to hundreds of thousands, without breaking the bank - our pricing plans include a generous number of projects within the monthly fee.
```
Comment (danieltprice) [inline content/use-cases/database-per-tenant.md:64]:
```suggestion
- You can scale your architecture progressively, from a few tenants to hundreds of thousands, without breaking the bank — our pricing plans include a generous number of projects within the monthly fee.
```

File: content/use-cases/database-per-tenant.md [hunk 4]
```diff
@@ -53,23 +46,42 @@ author={{
...
+If you’re building a B2B SaaS platform, a database-per-tenant design can simplify your architecture while preserving scalability. With Neon, when you place its tenant on its own project, you offer complete data privacy to your customers via instance-level isolation. This approach also makes it easy to comply with data regulations across different regions, as projects can be created in specific locations to meet local requirements.
 
-Additionally, each tenant can be scaled independently, optimizing both performance and costs while reducing operational risk. In the event of an issue, you can [run point-in-time recovery (PITR) for a specific tenant without impacting the entire fleet](https://neon.tech/docs/guides/branch-restore).
+Each tenant can be scaled independently, optimizing both performance and costs while reducing operational risk. And in the event of an issue / a customer request, you can [run point-in-time recovery (PITR) instantaneously for a specific tenant, without impacting the rest of the fleet](https://neon.tech/docs/guides/branch-restore).
```
Comment (danieltprice) [inline content/use-cases/database-per-tenant.md:80]:
```suggestion
Each tenant can be scaled independently, optimizing both performance and costs while reducing operational risk. And in the event of an issue or a customer request, you can [run point-in-time recovery (PITR) instantaneously for a specific tenant, without impacting the rest of the fleet](https://neon.tech/docs/guides/branch-restore).
```

File: content/use-cases/database-per-tenant.md [hunk 5]
```diff
@@ -53,23 +46,42 @@ author={{
...
 
-If you’re building a developer platform, [you can start offering Neon databases to your users](https://neon.tech/partners) via our Partnership plans. Neon is a cost-effective solution to power hobby plans / free tiers, also supporting your Enterprise customers by scaling seamlessly with their usage. Companies like [Vercel](https://neon.tech/blog/neon-postgres-on-vercel), [Replit](https://neon.tech/blog/neon-replit-integration), and [Koyeb](https://www.koyeb.com/blog/serverless-postgres-public-preview) are already using Neon to offer Postgres to their end-users.
+If you’re instead building a developer platform including a backend, or an [AI Agent](https://neon.tech/use-cases/ai-agents), you can start offering Neon databases to your users by becoming a [Partner](https://neon.tech/partners). Neon is a cost-effective solution to at the same time support your hobby plan and your Enterprise customers. Companies like [Vercel](https://neon.tech/blog/neon-postgres-on-vercel), [Replit](https://neon.tech/blog/neon-replit-integration), and [Koyeb](https://www.koyeb.com/blog/serverless-postgres-public-preview) are already using Neon to offer Postgres to their end-users.
```
Comment (danieltprice) [inline content/use-cases/database-per-tenant.md:84]:
```suggestion
If you’re instead building a developer platform including a backend, or an [AI Agent](https://neon.tech/use-cases/ai-agents), you can start offering Neon databases to your users by becoming a [Partner](https://neon.tech/partners). Neon is a cost-effective solution that can support your hobby plan and Enterprise customers at the same time. Companies like [Vercel](https://neon.tech/blog/neon-postgres-on-vercel), [Replit](https://neon.tech/blog/neon-replit-integration), and [Koyeb](https://www.koyeb.com/blog/serverless-postgres-public-preview) are already using Neon to offer Postgres to their end-users.
```

**Review by danieltprice — APPROVED** [review]

**Comment by saimonkat** [comment]
@carlotas19 sorry for the late reply here

All good, just noticed that this dash without spaces and brackets around sign up link looks like a bug
image

I just pushed the fix, please check https://github.com/neondatabase/website/pull/2423

**Pull request merged**